src/Fantomas/CodePrinter.fs

```diff
@@ -29,15 +29,18 @@ type ASTContext =
       /// First type param might need extra spaces to avoid parsing errors on `<^`, `<'`, etc.
       IsFirstTypeParam: bool
       /// Inside a SynPat of MatchClause
-      IsInsideMatchClausePattern: bool }
+      IsInsideMatchClausePattern: bool
+      /// Is a call to base class constructor
+      IsBaseCtorCall: bool }
```

Comment:
Please do not extend ASTContext, that is last resort solution that should always be avoided.
This has lead to many bugs and can be avoided here.